spec/models/cart_spec.rb

```diff
@@ -136,5 +136,27 @@
       expect(email_recipients).to eq(['requester@some-dot-gov.gov'])
     end
   end
+  describe '#restart' do
```

Comment:
Should restarting a cart always remove properties? Right now it's up to the caller to update the properties.